book/src/backup.md

```diff
@@ -0,0 +1,206 @@
+# Backups
+
+UNIT3D offers built in backup tools, available through the web dashboard or via Artisan commands, allowing you to create, manage, and restore your application snapshots.
+
+> [!IMPORTANT]  
+> Not recommended for large sites or when you need deduplication or distributed storage, as it may cause timeouts and performance issues.
```

Comment:
This is not necessarily true. Running the commands manually and the automatic runs daily via cron work fine on large sites. It's only an issue trying to run backup from Backup Manager in Staff Dashboard that timeouts for very large backups will occur.